CONTRIBUTING.md

```diff
@@ -99,3 +116,24 @@ Hooks.on("aipSetup", (packageConfig) => {
     packageConfig.push(config);
 });
 ```
+
+### Updating Event Listeners of an Application
+
+Here is an example of how updating the event listeners of an application could look like, for example after dynamically
+adding additional fields that match the selector of one of the field configs.
+
+```js
+function addAdditionalFields(activeEffectConfig) {
```

Comment:
Some pseudo jsdoc could help to make it more clear that `activeEffectConfig` should be a reference to the FormApplication.

In my own implementation it was just "this" as a parameter for the `registerFieldConfigs` call.

Reply:
done